openapi/openapi/src/main/scala/endpoints4s/openapi/model/OpenApi.scala

```diff
@@ -1,21 +1,22 @@
 package endpoints4s.openapi.model
 
-import java.io.Serializable
-
-import endpoints4s.{Encoder, Hashing}
 import endpoints4s.algebra.{ExternalDocumentationObject, Tag}
+import endpoints4s.{Encoder, Hashing}
 
+import java.io.Serializable
 import scala.collection.mutable
 
 /** @see [[https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md]]
   * @note Throws an exception on creation if several tags have the same name but not the same other attributes.
   */
 final class OpenApi private (
     val info: Info,
-    val paths: Map[String, PathItem],
+    private val innerPaths: collection.Map[String, PathItem],
     val components: Components
 ) extends Serializable {
```

Comment:
We have to add the following filter in `openapi/build.sbt`, in the settings of the `openapi` project:

~~~ scala
mimaBinaryIssueFilters ++= Seq(
  ProblemFilters.exclude[IncompatibleMethTypeProblem]("endpoints4s.openapi.model.OpenApi.this")
)
~~~

See https://stackoverflow.com/questions/31010755/scala-is-not-creating-private-constructor-for-companion-class

Reply:
Does it matter where in the file this is added? (for example within the openApi val, or is at the end of the file fine?)